sway/desktop/output.c

```diff
@@ -848,6 +871,14 @@ static void handle_commit(struct wl_listener *listener, void *data) {
 
 		update_output_manager_config(output->server);
 	}
+
+	if (event->committed & (WLR_OUTPUT_STATE_MODE |
+			WLR_OUTPUT_STATE_TRANSFORM |
+			WLR_OUTPUT_STATE_SCALE)) {
+		wlr_damage_ring_set_bounds(&output->damage_ring,
+			output->width, output->height);
```

Comment:
It would be slightly more convenient to make `wlr_damage_ring_set_bounds` return a `bool` indicating whether the bounds were updated or not.

Or maybe add `wlr_output_event_commit_has_size_changed()`?

Reply:
`wlr_damage_ring_set_bounds()` returning a `bool` makes sense to me.